Dynatrace/dynatrace-operator
# Update image labels in `Dockerfile`

# Description

Update outdated image labels.

## How can this be tested?
Verify labels are set correctly on the iamge
```
docker inspect docker.io/dynatrace/dynatrace-operator:<tag> --format='{{json .Config.Labels}}' | jq
```


## Checklist
